freqtrade/plot/plotting.py

```diff
@@ -158,12 +159,15 @@ def add_profit(fig, row, data: pd.DataFrame, column: str, name: str) -> make_sub
 
 
 def add_max_drawdown(fig, row, trades: pd.DataFrame, df_comb: pd.DataFrame,
-                     timeframe: str) -> make_subplots:
+                     timeframe: str, starting_balance: number) -> make_subplots:
```

Comment:
I think the proper typehint for this (and the other functions) is float, not number (this will also avoid importing Numpy just for typehints).